src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.csproj

```diff
@@ -12,9 +12,8 @@
     <TargetFrameworkIdentifier>.NETPortable</TargetFrameworkIdentifier>
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <UseCommonOutputDirectory>True</UseCommonOutputDirectory>
-    <!-- Don't transitively copy output files, since everything builds to the same folder. -->
     <CopyNuGetImplementations>false</CopyNuGetImplementations>
```

Comment:
The policy is that projects never include these elements.  I'm working on BuildBoss which will make this policy law.